models/governance/optimism/governance_schema.yml

```diff
@@ -0,0 +1,227 @@
+        name: start_date
+        description: "The date and time when the proposal voting started"
+
+      - &end_block
+        name: end_block
+        description: "The block timestamp for proposal voting end date"
+
+      - &end_date
+        name: end_date
+        description: "The date and time when the proposal voting ended"
+
+      - &platform
+        name: platform
+        description: "The decentralized governance platform on which the proposal was created (e.g., Agora, Snapshot)"
+
+      - &voters
+        name: voters
+        description: "The unique identifier of the voters who voted on particular proposals"
+
+      - &corresponding_voting_weightage
+        name: corresponding_voting_weightage
+        description: "The corresponding voting weight of the voter's"
+
+      - &corresponding_choices
+        name: corresponding_choices
+        description: "voter's voting choices(corresponding) for the proposal"
+
+      - &corresponding_choices_name
+        name: corresponding_choices_name
+        description: "voter's voting choice's name(corresponding) for the proposal"
+
+      - &highest_weighted_vote
+        name: highest_weighted_vote
+        description: "The vote with the highest weight"
+
+      - &highest_weighted_voter
+        name: highest_weighted_voter
+        description: "The voter whose vote is the highest weight"
+
+      - &highest_weighted_voter_percentage
+        name: highest_weighted_voter_percentage
+        description: "The percentage of the total voting weight that the highest weight voter has"
+
+      - &total_for_votingWeightage
+        name: total_for_votingWeightage
+        description: "The total voting weight of all the votes in favor of the proposal"
+
+      - &total_abstain_votingWeightage
+        name: total_abstain_votingWeightage
+        description: "The total voting weight of all the votes that abstained on the proposal"
+
+      - &total_against_votingWeightage
+        name: total_against_votingWeightage
+        description: "The total voting weight of all the votes against the proposal"
+
+      - &unique_for_votes
+        name: unique_for_votes
+        description: "The unique number of votes in favor of the proposal"
+
+      - &unique_abstain_votes
+        name: unique_abstain_votes
+        description: "The unique number of votes that abstained on the proposal"
+
+      - &unique_against_votes
+        name: unique_against_votes
+        description: "The unique number of votes against the proposal"
+
+      - &unique_votes_count
+        name: unique_votes_count
+        description: "The unique number of votes cast on the proposal"
+
+      - &total_votes_casted
+        name: total_votes_casted
+        description: "The total number of votes cast on the proposal, excluding abstaining votes"
+
+      - &proposal_status
+        name: proposal_status
+        description: "The current status of the proposal"
+
+  - name: governor_v5_proposals

```

Comment:
think v2 should also be included?

Reply:
All the proposals of v2 are in v5 as per my understanding. Agora shows 38 proposals on FrontEnd and all are in v5.